apps/core/migrations/0006_footertext.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 4.1.2 on 2022-10-21 14:52
+
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+import wagtail.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("wagtailcore", "0077_alter_revision_user"),
```

Comment:
Depending om `wagtailcore.0077` gives me: `ValueError: Related model 'wagtailcore.pagerevision' cannot be resolved` when running the tests.

Not sure why we need this dep.

Reply:
Yeah that was autogenerated with `makemigrations`, I think it should be safe to remove. It does highlight an issue with wagtail-localize though (https://github.com/wagtail/wagtail-localize/issues/642), which otherwise I wouldn't have found.

However I'm thinking of removing this entirely in favour of #177.

Reply:
I had to debug for quite a while to find this dep was the issue. 

Remove the footer logic, or not, I'll leave up to you! FWIW, your work seems more complete then #177

Reply:
Oh I mean just the footer model, not the PR itself. The snippet models in #177 also covers the three footer items as well, while mine only covers the copyright text.